luigi_pipeline/seqr_loading.py

```diff
@@ -109,6 +135,17 @@ def validate_mt(mt, genome_version, sample_type):
         :param sample_type: WGS or WES
         :return: True or Exception
         """
+        if genome_version == '37':
+            contig_check_result = contig_check(mt, GRCh37_STANDARD_CONTIGS, VARIANT_THRESHOLD)
+        elif genome_version == '38':
+            contig_check_result = contig_check(mt, GRCh38_STANDARD_CONTIGS, VARIANT_THRESHOLD)
+
+        if bool(contig_check_result):
```

Comment:
its fine to leave this as is, but just so you know python will check the "truthyness" of anything you provide to an `if` clause without explicitly casting it to a boolean. So empty dicts and lists will be treated as false, while dicts and lists with anything in them will be treated as true

Reply:
Yes that make sense. If the VCF passes the contig check, `contig_check()` will return an empty dict. Therefore, it won't raise any SeqrValidationErrors.